client/components/stats-date-control/stats-date-control-picker.tsx

```diff
@@ -77,8 +77,14 @@ const DateControlPicker = ( {
 				position="bottom"
 				context={ infoReferenceElement?.current }
 				isVisible={ popoverOpened }
+				className="stats-date-control-picker__popover-wrapper"
 			>
 				<div className="stats-date-control-picker__popover-content">
+					<DateControlPickerShortcuts
```

Comment:
Can I know more about why we are changing the order here and set direction to `row-reverse`? Thanks

Reply:
I see https://github.com/Automattic/wp-calypso/pull/83336/commits/09902bf7d05a20da3e7ca1efce5e767452c647b6